.github/workflows/tf-sandbox-devis.yml

```
@@ -0,0 +1,24 @@
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "sandbox/Devis/*"
+  push:
+    branches:
+      - main
+    paths:
+      - "sandbox/Devis/*"
+
+name: Devis-Sandbox
```

Workflow name should be similar to the folder path.. 
```suggestion
name: sandbox-devis
```